pkg/konfluxgen/integration-test-scenario.template.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: appstudio.redhat.com/v1beta2
+kind: IntegrationTestScenario
+metadata:
+  name: {{{ truncate ( sanitize .ApplicationName ) }}}-enterprise-contract
+  namespace: ocp-serverless-tenant
```

Comment:
As we configure the namespace anyhow, before we apply the manifests (see [here](https://github.com/openshift-knative/hack/blob/7587057e3c32be19f860767c26b3cbf20b2aea25/.github/workflows/apply-konflux-manifests.yaml#L31)), we could remove this here. Probably makes switching to another namespace easier too

Reply:
ok so I should remove the namespace: ocp-serverless-tenant from there, right? @creydr

Reply:
I'd do so